doc/radosgw/adminops.rst

```diff
@@ -347,7 +347,9 @@ Request Parameters
 :Type: String
 :Example: ``foo_user``
 :Required: Yes
-A tenant name may also specified as a part of ``uid``, by following the syntax ``tenant$user``, refer to `Multitenancy`_ for more details.
+
+A tenant name may also specified as a part of ``uid``, by following the syntax
+``tenant$user``, refer to `Multitenancy`_ for more details.
```

Comment:
Yes. The field list is what ended on line 349 with:

    :Required: Yes

Before, there was no newline, so Sphinx thought that the next line (starting with `A tenant...`) was part of the "field list" which was not the case. The newline separating the description from `Required` made the warning go away.